nrcan_etl/tests/embedded/test_walls.py

```diff
@@ -1,5 +1,8 @@
+import pytest
+
```

Comment:
should this blank line be there?

Reply:
I sometimes do that to separate the three sections of imports (standard, 3rd party, local package).

So yes intentional, but also fine with removing it